web/client-backplane/src/main/java/io/deephaven/javascript/proto/dhinternal/io/deephaven/proto/console_pb/figuredescriptor/ChartDescriptor.java

```diff
@@ -1700,6 +1726,8 @@ public static native ChartDescriptor.ToObjectReturnType toObject(
 
     public native JsArray<MultiSeriesDescriptor> getMultiSeriesList();
 
+    public native int getRow();
```

Comment:
did you manually set these?

Reply:
I generated them, but changed to `int` from `double` as that appears to be what colSpan did?

Reply:
As discussed I updated `integer_entitites.txt` on your branch and regenerated